config/initializers/zoo_stream.rb

```diff
@@ -2,9 +2,9 @@
   require 'aws-sdk'
   require 'zoo_stream'
   Aws.config.update region: 'us-east-1'
-  ZooStream.source = "talk-#{ Rails.env }"
+  ZooStream.source = ENV.fetch("ZOO_STREAM_SOURCE") || "talk"
```

Comment:
`ENV.fetch` will raise when the key doesn't exist, and I don't expect ENV values to ever by nil (at most an empty string, but not nil) so that or-statement won't have much of an effect.